packages/core/src/card/Card.css

```diff
@@ -1,23 +1,27 @@
 /* Styles applied to the root element */
 .saltCard {
+  --card-borderColor: var(--salt-container-primary-borderColor);
+  --card-color: inherit;
+  --card-shadow: var(--salt-overlayable-shadow);
+  --saltText-color: var(--card-color);
```

Comment:
Is this Card's responsibility?

Reply:
it's intended from design because of the disabled styling, and i did it like this since text would be passed via children - however, very true it should anyway be the responsibility of the consuemr to add the disabled variant to text, since card doesn't contain text necessarily - i just thought this would be an 100% way to ensure it. but i'm going to take it out now that i've reconsidered